### PR TITLE
Review

### PR DESCRIPTION
**Pull request by m0h1t**
Issue #1

### CODE REVIEW

**Comment by m0h1t** [comment]
Code refactoring will be done in issue #2

File: src/styles/css/App.css.map [hunk 1]
```diff
@@ -1 +1 @@
-{"version":3,"sourceRoot":"","sources":["../scss/Base/_base.fonts.scss","../scss/Base/_base.palette.scss","../scss/Base/_base.theme.scss","../scss/Base/_base.scss","../scss/Base/_base.settings.scss"],"names":[],"mappings":"AAGQ;AAER;EACE;;;AAGF;EACE;;;ACVF;AAEA;AAOA;ACTA;EACE,YDaO;ECZP,ODUO;;;AENT;EAEE,aCqBmB","file":"App.css"}
\ No newline at end of file
+{"version":3,"sourceRoot":"","sources":["../scss/Base/_base.settings.scss","../scss/Base/_base.fonts.scss","../scss/Base/_base.palette.scss","../scss/Base/_base.theme.scss","../scss/App.scss"],"names":[],"mappings":"AAMQ;AAAA;AAAA;ACJR;EACE,aD2BiB;;;ACxBnB;EACE,aDwBiB;;;AE/BnB;AAEA;AAOA;ACTA;EACE,YDaO;ECZP,ODUO;;;AEVT;EACE,YFWO;EEVP,OFQO;;;AELT;EACE;EACA,WJgBW;EIfX","file":"App.css"}
```
Comment (Shrikant9) [inline src/styles/css/App.css.map:1]: Make sure to add this file in `.gitignore` next time.

File: src/components/statusBar.jsx [hunk 2]
```diff
@@ -1,11 +1,38 @@
...
+    debugger;
+    let classes = 'col';
+    classes +=
+      this.props.currentTurn === player ? ' currentTurn' : ' inactivePlayer';
+    return classes;
+  };
+
+  _getResult = () => {
+    const { winner } = this.props;
+    if (winner === '') {
```
Comment (Shrikant9) [inline src/components/statusBar.jsx:15]: Refactor this to use JSON object instead of IF-ELSE.

**Review by Shrikant9 — APPROVED** [review]

**Comment by Shrikant9** [comment]
@m0h1t Take care of the comments in #2 .

File: src/components/statusBar.jsx (hunk 2 above)
Comment (m0h1t) [reply src/components/statusBar.jsx:15]: Resolved by using ternary operation instead of if{} else{} block

**Pull request merged**